packages/ListBox/src/components/Box/Box.styles.js

```diff
@@ -1,22 +1,26 @@
 import styled, { css } from "styled-components";
+import stylers from "@paprika/stylers";
+import tokens from "@paprika/tokens";
 
-const isInlineCSS = `
-  box-shadow: 0 0 0;
+const isInlineCSS = css`
+  border-top: 0;
   border-top-left-radius: 0;
   border-top-right-radius: 0;
-  border-top: 0;
+  box-shadow: 0 0 0;
 `;
 
 export const Box = styled.div(
-  ({ isInline, triggerWidth }) => css`
-    background-color: #ffffff;
-    border: 1px solid #d7d7d7;
-    border-radius: 3px;
-    box-shadow: 0px 1px 5px rgba(0, 0, 0, 0.07), 0px 7px 17px rgba(0, 0, 0, 0.1);
+  ({ hasError, isInline, isReadOnly, triggerWidth }) => css`
+    background-color: ${tokens.color.white};
+    border: 1px solid ${tokens.border.color};
+    border-radius: ${tokens.border.radius};
+    box-shadow: ${tokens.shadow};
     box-sizing: border-box;
     padding: 0;
-
-    ${isInline && isInlineCSS}
+    transition: border-color 0.2s;
```

Comment:
curious what does this do?

Reply:
There's a gentle transition when an error (red border) is applied.  There already was one for the trigger, so the transition looked a bit weird when it only applied to part of the outline.

<img width="405" alt="Screen Shot 2021-02-24 at 2 25 08 PM" src="https://user-images.githubusercontent.com/14944896/109074091-20b33500-76ac-11eb-980e-ed09f1ae1283.png">